auto_nag/scripts/regression_set_status_flags.py

```diff
@@ -51,6 +54,10 @@ def get_extra_for_template(self):
         return [f"esr{self.versions['esr_previous']}", f"esr{self.versions['esr']}"]
 
     def handle_bug(self, bug, data):
+        pc = f"{bug['product']}::{bug['component']}"
```

Comment:
Couldn't we add the component exceptions to the query itself? This way we don't needlessly get these bugs

Reply:
I thought about that too but in case we've more exceptions it could be make the query too big.
And I supposed that those bugs shouldn't represent that much (but I've no idea in fact).